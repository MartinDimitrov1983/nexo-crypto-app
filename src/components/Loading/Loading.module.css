.loadingContainer {
    display: flex;
    align-items: center;
    justify-content: center;
    position: absolute;
    top: 0;
    left: 0;
    height: 100%;
    width: 100%;
    z-index: 1;
    backdrop-filter: blur(4px);
}

.loadingCircle {
    border: 8px solid rgb(243, 243, 243);
    border-top: 8px solid rgb(52, 152, 219);
    border-radius: 50%;
    width: 50px;
    height: 50px;
    animation: spin 1s linear infinite;
}

@keyframes spin {
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
}
