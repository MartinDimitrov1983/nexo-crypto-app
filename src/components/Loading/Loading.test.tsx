import React from 'react';
import { render } from '@testing-library/react';

import Loading from './Loading';

test('Snapshot Loading', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
});
