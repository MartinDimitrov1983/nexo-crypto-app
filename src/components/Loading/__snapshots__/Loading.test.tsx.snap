// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Loading 1`] = `
<DocumentFragment>
  <div
    class="loadingContainer"
  >
    <div
      class="loadingCircle"
    />
  </div>
</DocumentFragment>
`;
